### ISSUE TITLE
Add Python 3.9.7 to VS 16.11, 16.9, 16.7, 15.9, then 17.0, 17.1, and main

### ISSUE THREAD

**Issue by AdamYoblick** [microsoft/PTVS#6758]
## 16.11

- [x] Investigate if installing a newer version of the python package (with the same package id) wipes out the old one.
       - Matt says it's supposed to, so we need a new package.
- [x] Mark Python 3.7 as out of support
          - See devdiv.visualstudio.com/DevDiv/_git/VS/commit/f8267b5a6507514146678297f2df36bda0d9fdff?refName=refs%2Fheads%2Fdev%2Fjudej%2Fd16.4-oos for an example
          - 3.7 should be UNCHECKED by default (optional)
          - **VERY IMPORTANT NOTE: When marking an explicitly versioned component as out of support, you MUST increase the version of the component, or else the VS installer won't even recognize that it's been changed.**
- [x] Add Python 3.9.7 to VS installer (new package id!)
          - See the [OneNote](https://microsoft.sharepoint.com/teams/python/_layouts/OneNote.aspx?id=%2Fteams%2Fpython%2FDocuments%2FDocuments%2FPython%20Team&wd=target%28Python%20VS%2FDev%20Docs.one%7CD7206CE2-CF40-437B-8CE9-1E55F4BC2F44%2FUpdate%20VS%20Bundled%20Python%20Version%7C2D123ECB-BEC2-4CA8-9597-19BD5F809205%2F%29) for instructions
          - 3.9 should be UNCHECKED by default (optional)
- [x] Merge PR into Visual Studio
          - https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/361805, just waiting on QB

16.11 release schedule is at https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/9727/Dev16.11-Schedule-(LTS). We're targeting 16.11.19, so this change needs to be MERGED INTO VS by December 1, 2022.

- [x] Test

## 16.9
- [x] Repeat for VS 16.9
          - VS PR is at https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/363143, just waiting on QB
- [x] Test

## 16.7
- [x] Repeat for VS 16.7
          - VS PR is at https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/363444, just waiting on QB
- [x] Test

## 15.9
- [x] Repeat for VS 15.9
          - VS PR is at https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/363936, just waiting on QB
- [x] Test (waiting on PR test failures that are not ours to fix, no build to test with)

## 17.0 and main (DUAL CHECK IN)
- [x] Repeat for VS 17.0
          - VS PR into 17.0 is at https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/364010
- [x] Test 17.0
- [x] Repeat for main 
          - VS PR into main is at https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/364028
- [x] Test main

**Comment by AdamYoblick** [comment]
All testing is complete, and PR into main is completed. All other PR's are waiting for QB approval.
